File: src/net/auoeke/limitless/config/anvil/AnvilConfiguration.java
package net.auoeke.limitless.config.anvil;

import net.auoeke.limitless.config.anvil.entry.AnvilNormalizationEntry;
import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject;

public class AnvilConfiguration {
    public boolean mergeConflicts = true;

    public boolean mergeIncompatible = false;

    public boolean fixedCost = true;

    public int levelLimit = Integer.MAX_VALUE;

    @CollapsibleObject
    public AnvilNormalizationEntry normalization = new AnvilNormalizationEntry();
}

File: src/net/auoeke/limitless/config/anvil/AnvilConfiguration.kt
package net.auoeke.limitless.config.anvil

import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject
import net.auoeke.limitless.config.anvil.entry.AnvilNormalizationEntry

class AnvilConfiguration {
    var mergeConflicts = true
    var mergeIncompatible = false
    var fixedCost = true
    var levelLimit = Int.MAX_VALUE

    @CollapsibleObject
    var normalization = AnvilNormalizationEntry()
}
